### PR TITLE
select: revert select's dropdown icon to default

### PR DESCRIPTION
**Pull request by Hyjaz**
screen shot 2018-06-02 at 2 12 14 pm

This is a fix for issue #103 

I removed the following three css properties from select html tag:

`-webkit-appearance:none;-moz-appearance:none;appearance:none`

### CODE REVIEW

**Comment by ivangabriele** [comment]
Where are the original Sass or CSS files from which this file has been generated ?

**Comment by Hyjaz** [comment]
For `<select />` it's [here](https://github.com/Electra-project/Electra-Styleguide/blob/master/src/assets/toolkit/styles/components/_dropdown.scss)

**Comment by ivangabriele** [comment]
It was not a bad idea. But how we do it seems wrong. We should have it as a git submodule or a npm dependency (a npm dependency being easier to handle imo) so that we have to update it to generate the css on this project.

In the meantime, can you add a PR on the Electra-Styleguide too ?

**Comment by Hyjaz** [comment]
The corresponding PR has been added on Electra-Styleguide - (#2)[https://github.com/Electra-project/Electra-Styleguide/pull/2]

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +26,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Mnemonic recovery.
 - Hidden normal quitting error (acting as an install-update-and-quit instead of a simple daemon closing).
 - Missing prefixes in (crypto)currencies top shown values.
+- Payments page caret down icon.
```
Comment (ivangabriele) [inline CHANGELOG.md]: It's not a change, it's a fix ;) Changed is for stuff that worked before and has been changed.

Comment (Hyjaz) [reply CHANGELOG.md]: it's under `Fixed` already

**Pull request merged**